File: 707_Tests/Roland/Roland/MainStarter.cs
using System;
using System.Windows.Forms;

namespace Roland
{
    public static class MainStarter
    {
        /// <summary>
        /// The main entry point for the application.
        /// </summary>
        [STAThreadAttribute]
        public static void Main()
        {
            Application.EnableVisualStyles();
            Application.SetCompatibleTextRenderingDefault(false);
            Application.Run(new FormMain());
        }
    }
}

File: 707_Tests/TestAffair_002/TestAffair_002/MainStarter.vb
Imports System.Windows.Forms

Public Module MainStarter

    Public Sub Main()
        Application.EnableVisualStyles()
        Application.SetCompatibleTextRenderingDefault(False)
        Dim t_hbf As New HB_Form()
        Application.Run(t_hbf)
    End Sub

End Module
